.github/workflows/licenses.yml

```
@@ -0,0 +1,40 @@
+name: Licenses
+
+on:
+  push:
+    branches:
+      - dev
+      - stable
+      - coordicide
+  pull_request:
+    branches:
+      - dev
+      - stable
+      - coordicide
+
+jobs:
+  format:
```

```suggestion
  licenses:
```
Probably copy-pasted 😬

---

you bet